lib/astutils.cpp

```diff
@@ -648,7 +648,8 @@ bool exprDependsOnThis(const Token* expr, nonneg int depth)
     return exprDependsOnThis(expr->astOperand1(), depth) || exprDependsOnThis(expr->astOperand2(), depth);
 }
 
-static bool hasUnknownVars(const Token* startTok) {
+static bool hasUnknownVars(const Token* startTok)
```

Comment:
and this is it a manual change? I am confused because I have been running astyle 3.0.1 myself and it does not do this.